src/ansys/tools/path/path.py

```diff
@@ -10,7 +10,7 @@
 
 from ansys.tools.path.misc import is_float, is_linux, is_windows
 
-LINUX_DEFAULT_DIRS = [["/", "usr", "ansys_inc"], ["/", "ansys_inc"], ["/", "install", "ansys_inc"]]
+LINUX_DEFAULT_DIRS = [["/", "usr", "ansys_inc"], ["/", "ansys_inc"]]
```

Comment:
why are you removing this? I see it is not a common path. Have you check who added it? and why?